LEGO1/lego/legoomni/src/race/carrace.cpp

```diff
@@ -1,60 +1,418 @@
 #include "carrace.h"
 
+#include "actions/carrace_actions.h"
+#include "dunebuggy.h"
+#include "isle.h"
+#include "legoanimationmanager.h"
+#include "legobackgroundcolor.h"
+#include "legocontrolmanager.h"
+#include "legohideanimpresenter.h"
+#include "legonavcontroller.h"
+#include "legopathstruct.h"
+#include "legoracers.h"
+#include "legoutils.h"
+#include "misc.h"
 #include "mxactionnotificationparam.h"
+#include "mxbackgroundaudiomanager.h"
+#include "mxmisc.h"
+#include "mxnotificationmanager.h"
+#include "mxstillpresenter.h"
+#include "mxtransitionmanager.h"
+#include "mxvariabletable.h"
+#include "scripts.h"
 
 DECOMP_SIZE_ASSERT(CarRace, 0x154)
 
+// GLOBAL: LEGO1 0x100d5d10
+const MxS32 CarRace::g_unk0x100d5d10[] = {
```

Comment:
I know `const` is generally the right qualifier here, but we haven't used it anywhere else so far for those global variables so I think I'd remove it simply for consistency - unless that worsens the match somewhere?

Reply:
I removed the `const`s and I didn't see any (significant) regressions, so it's likely fine. Thanks!